client/routes/MobileNav.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { Button, Dropdown, Icon, Menu } from 'semantic-ui-react';
+
+import Session from '@client/util/session';
+
+Session.timeout();
+
+const MobileNav = () => {
+    const [menuExpanded, setMenuExpanded] = useState(false);
+    return (
+        <React.Fragment>
+            <Button
+                icon
+                fluid
+                labelPosition="left"
+                basic
+                color="black"
```

Comment:
I'm not sure why these buttons have the dark black border—i've always wondered about that that 🤷‍♂ It's especially strange now that menu is default on desktop and then has this harsh black border on mobile. Can we just ditch hard-coded color on buttons and leave them default?

Reply:
I like it, but don't care particularly strongly enough to say they should be black. I had also worried about color contrast with Semantic UI's default button styling, but I just checked it and it seems to be fine according to the WAVE accessibility extension. We can have it be the default if you want.